packages/devextreme/testing/testcafe/tests/dataGrid/fixedColumns.ts

```diff
@@ -319,3 +322,214 @@ test('Hovering over a row should work correctly after scrolling when there is a
 
   await makeRowsViewTemplatesAsync(DATA_GRID_SELECTOR, 100);
 });
+
+test('Accessibility: Scrollable should always have focusable element when navigate through grid', async (t) => {
+  const dataGrid = new DataGrid('#container');
+
+  const columnsByFixing = await ClientFunction(
+    () => (window as any).widget.getVisibleColumns().map((column) => column.fixed),
+  )();
+  const columnsLength = await ClientFunction(
+    () => (window as any).widget.getVisibleColumns().length,
+  )();
+
+  const pressKey = async (key: string, times = 1) => {
+    for (let i = 0; i < times; i += 1) {
+      await t.pressKey(key);
+    }
+  };
```

Comment:
Let's move this helper function on top of the test file.
As far as I can see, it used in other test cases too (duplicates)